lib/Backend/arm/LowerMD.cpp

```diff
@@ -9295,4 +9295,15 @@ LowererMD::LowerTypeof(IR::Instr* typeOfInstr)
     typeOfInstr->InsertBefore(helperLabel);
     typeOfInstr->InsertAfter(doneLabel);
     m_lowerer->LowerUnaryHelperMem(typeOfInstr, IR::HelperOp_Typeof);
-}
\ No newline at end of file
+}
+
+#if DBG
+//
+// Helps in debugging of fast paths.
+//
+void LowererMD::GenerateDebugBreak( IR::Instr * insertInstr )
+{
+    IR::Instr *int3 = IR::Instr::New(Js::OpCode::DEBUGBREAK, insertInstr->m_func);\
```

Comment:
The naming as "GenerateDebugBreak" is to have it match up with the name of the x86/amd64 version at LowerMDShared.cpp:7897. The reason that this function needed to be added here was that while the code added to Lower.cpp built on x86 and amd64, it didn't build on ARM due to this symbol not being defined in the platform's LowerMD. I've removed the trailing '\\'; it was due to copying the break generation from an arm-specific preprocessor macro.